modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -347,9 +347,10 @@ const Select = ({
 		}
 	};
 
-	const inputTrigger = document.querySelector(
+	const inputTrigger = document.querySelectorAll(
 		'.lfr__ddm-select-input-trigger'
-	);
+	)[document.querySelectorAll('.lfr__ddm-select-input-trigger').length - 1];
+
```

Comment:
Hey, @ericthmoritsuka! Thanks for working on this issue! I see you are trying to select a specific element with a ".lfr__ddm-select-input-trigger " class, but doing that by getting the lest element in the node list is not the safest way of doing it. We may have cases when this element is not the last one!

I suggest you use the child combinator selector and select the parent by attribute value, as bellow: 

```
const inputTrigger = document.querySelector(
	'[data-field-reference|="predefinedValue"] > .lfr__ddm-select-input-trigger'
);
```

Reply:
Hi Carlos, thanks for the review! I thought this fix would not be ideal and I was wondering what could be done to make it work the right way. Thank you for the suggestion. I will work on it and get back to you.